src/components/TokenInput/TokenInput.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Button } from '@pancakeswap-libs/uikit'
+import { Button, Flex, Input, InputProps } from '@pancakeswap-libs/uikit'
 import useI18n from '../../hooks/useI18n'
-import Input, { InputProps } from '../Input'
 
 interface TokenInputProps extends InputProps {
   max: number | string
   symbol: string
+  value: string
   onSelectMax?: () => void
+  onChange?: (evt: React.FormEvent<HTMLInputElement>) => void
```

Comment:
Additionally, why are we typing this? It is already included in `InputProps`...

Reply:
The ui-kit InputProps type only has scale, isSuccess and is isWarning props.